v1.0/known-limitations.md

```diff
@@ -28,6 +28,36 @@ Within a single [transaction](transactions.html):
 - A [`CREATE TABLE`](create-table.html) statement containing [`FOREIGN KEY`](foreign-key.html) or [`INTERLEAVE`](interleave-in-parent.html) clauses cannot be followed by statements that reference the new table.
 - A table cannot be dropped and then recreated with the same name. This is not possible within a single transaction because `DROP TABLE` does not immediately drop the name of the table. As a workaround, split the [`DROP TABLE`](drop-table.html) and [`CREATE TABLE`](create-table.html) statements into separate transactions.
 
+## Schema changes between executions of prepared statements
+
+When the schema of a table targeted by a prepared statement changes before the prepared statement is executed, CockroachDB should return an error. Instead, CockroachDB incorrectly allows the prepared statement to return results based on the changed table schema, for example:
```

Comment:
Should this really be an error? The trace below looks reasonable to me. We should add a best-practice warning against using `SELECT *` for queries that will be repeated (whether via prepared statements or not), but I don't think we need to document this as a limitation. (We might want to say that this is *deprecated* if we plan to change this behavior in the future as we do more caching of query plans or output schemas). 

The insert/update cases below are appropriate to document as known limitations.

Reply:
Thanks, @bdarnell. I've left this in, for now, and reworded it to simply explain that this can happen and propose that recommendation. Post-release, I can work on moving it somewhere else.